t/t2612-job-shell-pty.t

```diff
@@ -136,7 +136,7 @@ nul_ctrl_d() {
 }
 test_expect_success 'pty: NUL (Ctrl-SPACE) character not dropped' '
 	nul_ctrl_d | flux run -vvv -o pty.interactive -o pty.capture cat -v &&
-	flux job eventlog -HLp output $(flux job last) &&
-	flux job eventlog -HLp output $(flux job last) | grep \\^@
+	flux job eventlog -HL -p output $(flux job last) &&
```

Comment:
This commit should probably go before 90f0d30 so the test isn't broken in between (and note in the commit message that `-L` will be fixed to take an optional arg so this usage will break)

Reply:
oh yeah! i fixed that last as it was uncovered.